content/body/switch.php

```diff
@@ -2,13 +2,13 @@
   A switch is like a checkbox, in that is designed to be an input control that has a binary value
   (either <strong>checked</strong> and <strong>unchecked</strong> or <strong>on</strong> or <strong>off</strong>,
   depending on the screen reader).
-  Like <a href="tabs.php">tablists.php</a>, switches do not have a native HTML5 tag, so we implement custom code using
+  Like <a href="tabs.php">tablists.php</a>, switches/Toggles do not have a native HTML5 tag, so we implement custom code using
   the
-  <code>switch</code> role in JavaScript.
+  <code>switch/toggle</code> role in JavaScript.
```

Comment:
There is no such thing as an ARIA toggle role, which is why it just said "switch" in this sentence.

```suggestion
  <code>switch</code> role in JavaScript.
```